inference-engine/tests/functional/plugin/shared/src/subgraph_tests/multiply_concat_reshape_reshape_concat.cpp

```diff
@@ -0,0 +1,57 @@
+#include "functional_test_utils/precision_utils.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+#include "subgraph_tests/multiply_concat_reshape_reshape_concat.hpp"
+
+namespace LayerTestsDefinitions {
+    std::string ConcatReshapeReshapeConcatMul::getTestCaseName(const testing::TestParamInfo<ConcatReshapeReshapeConcatMulTuple> &obj) {
+        std::vector<std::vector<size_t>> input;
+        InferenceEngine::Precision netPrecision;
+        std::string targetName;
+        std::tie(input, netPrecision, targetName) = obj.param;
+        std::ostringstream results;
+
+        results << "IS=" << CommonTestUtils::vec2str(input[0]) << "_";
+        results << "netPRC=" << netPrecision.name() << "_";
+        results << "targetDevice=" << targetName << "_";
+        return results.str();
+    }
+
+    void ConcatReshapeReshapeConcatMul::SetUp() {
+        std::vector<std::vector<size_t>> inputs;
+        InferenceEngine::Precision netPrecision;
+        std::tie(inputs, netPrecision, targetDevice) = this->GetParam();
+        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+        auto input = ngraph::builder::makeParams(ngPrc, {inputs});
+
+        auto const_concat = std::make_shared<ngraph::opset1::Constant>(ngPrc,
+                                                                 ngraph::Shape{input[0]->get_shape()},
+                                                                std::vector<size_t>{1});
+        auto concat = std::make_shared<ngraph::opset1::Concat>(ngraph::OutputVector{input[0]->output(0),
+                                                                                    const_concat->output(0)},
+                                                                                            0);
+        auto squeeze = ngraph::builder::makeUnsqueeze(concat, ngraph::element::i64, std::vector<size_t>{0});
+        auto squeeze2 = ngraph::builder::makeSqueeze(squeeze, ngraph::element::i64, std::vector<size_t>{0});
+        auto const_concat2 = std::make_shared<ngraph::opset1::Constant>(ngPrc,
+                                                                       ngraph::Shape{squeeze2->get_shape()},
+                                                                       std::vector<size_t>{1});
+        auto concat2 = std::make_shared<ngraph::opset1::Concat>(ngraph::OutputVector{squeeze2->output(0),
+                                                                                    const_concat2->output(0)},
+                                                               0);
+        auto const_mul = ngraph::builder::makeConstant(ngPrc, ngraph::Shape{1}, std::vector<float>{-1.0f});
+        auto mult = std::make_shared<ngraph::opset1::Multiply>(input[0], const_mul);
+        function = std::make_shared<ngraph::Function>(mult, input, "concat_reshape_reshape_concat_mul");
+    }
+
+    TEST_P(ConcatReshapeReshapeConcatMul, CompareWithRefs){
```

Comment:
why we are not creating unit tests at all? Functional test to me only should happen if fight with accuracy. And event than it is questionable.